modules/pico_tcp.c

```diff
@@ -54,12 +54,16 @@
 
 
 #ifdef PICO_SUPPORT_TCP
-#define tcp_dbg_nagle(...) do {} while(0)
-#define tcp_dbg_options(...) do {} while(0)
-
 
-#define tcp_dbg(...) do {} while(0)
-/* #define tcp_dbg dbg */
+#ifdef DEBUG_TCP
+#define tcp_dbg_nagle        dbg
+#define tcp_dbg_options      dbg
+#define tcp_dbg              dbg
```

Comment:
would it make more sense to have 3 separate `#ifdef`s here? Looks like they're probably different levels of debugging. Wouldn't make much sense to separate them out, and then always enable or disable them in group.